test/fake-server.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+var express = require('express'),
+    bodyParser = require('body-parser'),
+    morgan = require('morgan'),
+    $port = 3030,
+    server;
```

Comment:
Prefer one statement per variable.

Declare `server` where it's first used if possible.

Use `PORT` for constants rather than `$port`.